### PR TITLE
Introduce formula for PDF Sandwich

### PR DESCRIPTION
**Pull request by whistlerbrk**
make / make install need to be separate or build will fail

### CODE REVIEW

**Comment by whistlerbrk** [comment]
Thank you @mistydemeo for help in #machomebrew

File: Library/Formula/pdfsandwich.rb [hunk 1]
```diff
@@ -0,0 +1,27 @@
+class Pdfsandwich < Formula
+  desc "Tool to make \"sandwich\" OCR pdf files"
+  homepage "http://www.tobias-elze.de/pdfsandwich/"
+  url "https://downloads.sourceforge.net/project/pdfsandwich/pdfsandwich%200.1.4/pdfsandwich-0.1.4.tar.bz2"
+  sha256 "8b82f3ae08000c5cae1ff5a0f6537b0b563befef928e5198255b743a46714af3"
+
+  depends_on "tesseract"
+  depends_on "ghostscript"
+  depends_on "imagemagick"
+  depends_on "ocaml"
```
Comment (DomT4) [inline Library/Formula/pdfsandwich.rb]: Runtime?

File: Library/Formula/pdfsandwich.rb [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  depends_on "gawk"
+
+  def install
+    bin.mkpath
+    system "./configure", "--disable-debug",
+                          "--disable-dependency-tracking",
+                          "--prefix=#{prefix}"
+
+    system "make"
+    system "make install"
```
Comment (DomT4) [inline Library/Formula/pdfsandwich.rb]: `system "make", "install"`

File: Library/Formula/pdfsandwich.rb [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+  end
+
+  test do
+    system "pdfsandwich"
```
Comment (DomT4) [inline Library/Formula/pdfsandwich.rb]: `system bin/"pdfsandwich"`

**Comment by DomT4** [comment]
Ping?

**Comment by whistlerbrk** [comment]
Sorry, gmail categorization's got me missing emails left and right. Will
look into this today

On Sunday, October 11, 2015, Dominyk Tiller <notifications@github.com>
wrote:

> Ping?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Homebrew/homebrew/pull/44045#issuecomment-147274681>.
>


-- 
Kunal Shah
kunalashah.com

**Comment by DomT4** [comment]
Closing as dead. Please feel free to resubmit as fresh with the comments above addressed and audit passing, if you'd like to. Thanks for the submission to Homebrew!

**Pull request closed without merging**